### DISCUSSION TITLE
致美化 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#20] [模板讨论]
致美化 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by infisz** [comment]
4月份开始失效了好像。
Failed at 1/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: OpenSSL SSL_read: Connection reset by peer, errno 104,
Request URL: https://zhutix.com/wp-json/b2/v1/getRecaptcha

**Reply by wjf0214** [reply]
`OpenSSL SSL_read: Connection reset by peer` 这可能是你本身网络问题，访问网站异常。我看了下自己的，没有问题。积分正常增长。

**Comment by miaoxiaozhao** [comment]
恭喜！您今天获得了【\"credit\":\"2\"】积分

格式可以优化一下

**Reply by wjf0214** [reply]
已优化。
